src/DynamoRevitInstall/Product.wxs

```diff
@@ -65,20 +83,23 @@
     </Property>
 
     <Property Id='DYNAMOREVIT_INSTALLDIR'></Property>
-    <SetProperty Id="DYNAMOREVIT_INSTALLDIR" Value="[DYNAMOLOCATION]" After="LaunchConditions"></SetProperty>
+    <SetProperty Id="DYNAMOREVIT_INSTALLDIR" Value="[ProgramFiles64Folder]\$(var.ProductName) $(var.Major).$(var.Minor)" After="LaunchConditions"></SetProperty>
 
-    <Feature Id="DYNAMOREVIT_CORE_FEATURE"
+    <Icon Id="DynamoInstaller.ico" SourceFile="$(var.base)\tools\install\Extra\DynamoInstaller.ico"/>
+    <Property Id="ARPPRODUCTICON" Value="DynamoInstaller.ico" />
+    <!--================-->
+    <Feature Id="DYNAMO_REVIT_FEATURE"
              Title="Dynamo for Revit"
              Level="1">
       <ComponentGroupRef Id="RELEASE"/>
-      <Feature Id="DYNAMO_CORE_SAMPLES"
+      <ComponentRef Id="ProductRegistry" />
+      <Feature Id="DYNAMO_REVIT_SAMPLES"
              Title="Sample Content"
              Level="1" >
         <ComponentGroupRef Id="SAMPLES"/>
       </Feature>
     </Feature>
 
-    <!--Revit features are only the addin. All Revit assets are currently installed as part of core.-->
     <Feature Id="DYNAMO_REVIT_2014"
```

Comment:
We don't support for Revit2014, we need to have components for 2015, 2016 and 2017.